docs/v1/api.md

```diff
@@ -1,43 +1,63 @@
 # V1 API
 
-This document contains the Kubernetes resource types for the v1 service catalog.
+This document contains resource types and their standard usage in v1 service
+catalog. Although this API will be implemented in Kubernetes, other systems
+are not precluded from implementing it as well.
 
 # Resource Types
 
 This section lists descriptions of the resources in the service catalog API.
 
-_ __Note:__ All names herein are tentative, and should be considered placeholders
+*__Note:__ All names herein are tentative, and should be considered placeholders
 and used as placeholders for purposes of discussion only. This note will be
-removed when names are finalized._
+removed when names are finalized.*
+
+## `Broker`
+
+This resource is created by an administrator to instruct the service catalog's
+controller event loop to do the following:
+
+1. Make a request against a given CF service broker's catalog endpoint
+   (`GET /v2/catalog`)
+2. Translate the response to a list of `ServiceClass`es
+3. Write each translated `ServiceClass` to stable storage
+
+*TODO: should we call out what happens when a `Broker` resource is deleted?*
 
 ## `ServiceClass`
 
-This resource indicates a general kind of backing service that a consumer
-may request. The 'service kind' concept is purposefully arbitrary. We expect
-each cluster operator team to assign specific meaning to the ones they choose.
+This resource is created by the service catalog's controller event loop after
+it has received a `Broker` resource and successfully called the backing CF
+broker's catalog endpoint. It represents a single (serviceID, planID) pair that
+a CF service broker has made available.
+
+*TODO: what happens in the below cases?*
 
-## `ServiceInstance`
+1. *The `Broker` that caused the `ServiceClass` was deleted*
+1. *The `ServiceClass` itself was deleted*
 
-This resource indicates that a request by a consumer for a usable `ServiceClass`
-has been successfully executed. Consumers may reference these resources to
-begin using the backing service it represents.
+## `Instance`
 
-## `ServiceClaim`
+This resource is created by a service consumer to indicate their desire to
+provision a service. When the service catalog's controller event loop sees an
+`Instance` created, it calls the provision endpoint on the backing CF service
+broker and writes `provisioned` into a `status` field of the
```

Comment:
I think brian grant had some commentary about this that we should not have a status field and instead interpret the current state from some other fields.

The status field **is** what we agreed on.